src/tests/Dictionary/DictionaryContainer.test.jsx

```diff
@@ -48,6 +48,67 @@ describe('DictionaryOverview', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  it('should handle HEAD version release', () => {
+    const props = {
+      dictionary: [dictionary],
+      versions: [customVersion],
+      dictionaryConcepts: [dictionary],
+      isReleased: false,
+      match: {
+        params: {
+          ownerType: 'testing',
+          owner: 'tester',
+          type: 'collection',
+          name: 'chris',
+        },
+      },
+      fetchDictionary: jest.fn(),
+      fetchVersions: jest.fn(),
+      fetchDictionaryConcepts: jest.fn(),
+      releaseHead: jest.fn(),
+    };
+    const wrapper = mount(<Provider store={store}><MemoryRouter><DictionaryOverview {...props} /></MemoryRouter></Provider>);
+    expect(wrapper.find('.fas.fa-cloud-upload-alt')).toHaveLength(1);
+    expect(wrapper).toMatchSnapshot();
+    wrapper.find('.fas.fa-cloud-upload-alt').simulate('click');
+  });
+
+  it('should handle componentWillUpdate in HEAD version release', () => {
+    const props = {
+      dictionary: [dictionary],
+      versions: [customVersion],
+      dictionaryConcepts: [dictionary],
+      isReleased: false,
+      match: {
+        params: {
+          ownerType: 'testing',
+          owner: 'tester',
+          type: 'collection',
+          name: 'chris',
+        },
+      },
+      fetchDictionary: jest.fn(),
+      fetchVersions: jest.fn(),
+      fetchDictionaryConcepts: jest.fn(),
+      releaseHead: jest.fn(),
+    };
+    const wrapper = mount(<Provider store={store}><MemoryRouter><DictionaryOverview {...props} /></MemoryRouter></Provider>);
+    const newProps = {
+      dictionary: [dictionary],
+      match: {
+        params: {
+          ownerType: 'testing',
+          owner: 'tester',
+          type: 'collection',
+          name: 'chris',
+        },
+      },
+      isReleased: true,
+      fetchVersions: jest.fn(),
+    };
+    wrapper.setProps(newProps);
```

Comment:
This test is not asserting anything. Please refactor accordingly.

Reply:
Thanks for noticing this. Will work on it.